nrf_security/doc/backend_config.rst

```diff
@@ -82,44 +52,44 @@ Feature support
-
-+--------------+-----------------+---------------------------------------------+----------------------------------------+
-| Cipher mode  | Support         | Configurations                              | Note                                   |
-+==============+=================+=============================================+========================================+
-| CTR          | Glue            | :kconfig:`CONFIG_MBEDTLS_CIPHER_MODE_CTR`   |                                        |
-+--------------+-----------------+---------------------------------------------+----------------------------------------+
-| CBC          | Glue            | :kconfig:`CONFIG_MBEDTLS_CIPHER_MODE_CBC`   |                                        |
-+--------------+-----------------+---------------------------------------------+----------------------------------------+
-| CFB          |                 | :kconfig:`CONFIG_MBEDTLS_CIPHER_MODE_CFB`   | Original Mbed TLS and nrf_oberon only  |
-+--------------+-----------------+---------------------------------------------+----------------------------------------+
-| OFB          |                 | :kconfig:`CONFIG_MBEDTLS_CIPHER_MODE_OFB`   | Original Mbed TLS and nrf_oberon only  |
-+--------------+-----------------+---------------------------------------------+----------------------------------------+
-| XTS          |                 | :kconfig:`CONFIG_MBEDTLS_CIPHER_MODE_XTS`   | Original Mbed TLS and nrf_oberon only  |
-+--------------+-----------------+---------------------------------------------+----------------------------------------+
+AES cipher modes can be configured by setting Kconfig variables in the following table:
+
++--------------+---------------------------------------------+----------------------------------------+
+| Cipher mode  | Configurations                              | Note                                   |
++==============+=============================================+========================================+
+| CTR          | :kconfig:`CONFIG_MBEDTLS_CIPHER_MODE_CTR`   |                                        |
++--------------+---------------------------------------------+----------------------------------------+
+| CBC          | :kconfig:`CONFIG_MBEDTLS_CIPHER_MODE_CBC`   |                                        |
++--------------+---------------------------------------------+----------------------------------------+
+| CFB          | :kconfig:`CONFIG_MBEDTLS_CIPHER_MODE_CFB`   | Original Mbed TLS and nrf_oberon only  |
++--------------+---------------------------------------------+----------------------------------------+
+| OFB          | :kconfig:`CONFIG_MBEDTLS_CIPHER_MODE_OFB`   | Original Mbed TLS and nrf_oberon only  |
++--------------+---------------------------------------------+----------------------------------------+
+| XTS          | :kconfig:`CONFIG_MBEDTLS_CIPHER_MODE_XTS`   | Original Mbed TLS and nrf_oberon only  |
```

Comment:
The user can't enable "Original Mbed TLS" without also using nrf_oberon. So it suffices to state

"nrf_oberon only"

Reply:
Please search of "Original" and remove references to "Original Mbed TLS" when it is OR'd or AND'd with nrf_oberon.

Reply:
Removed Original Mbed TLS from the tables, and updated the comments to **nrf_oberon only**